nixos/modules/services/databases/postgresql.nix

```diff
@@ -168,6 +168,18 @@ in
                 }
               '';
             };
+            ensureRoles = mkOption {
+              type = types.listOf (types.enum [ "superuser" "create role" "create db" "replication" "bypass urls" ]);
+              default = [ ];
+              description = ''
```

Comment:
```suggestion
              description = lib.mdDoc ''
```

Reply:
Oh, thanks for the suggestion! I am happy to convert the rest of the descriptions to `mdDoc` as well, but didn't want to add too much noise to this pr.